build-support/bin/release.sh

```diff
@@ -596,16 +594,27 @@ function build_pex() {
     platform_flags=("${platform_flags[@]}" "--platform=${platform}")
   done
 
-  pex \
-    -o "${dest}" \
-    --entry-point="pants.bin.pants_loader:main" \
-    --no-build \
-    --no-pypi \
-    --disable-cache \
-    "${platform_flags[@]}" \
-    -f "${DEPLOY_PANTS_WHEEL_DIR}/${PANTS_UNSTABLE_VERSION}" \
-    -f "${DEPLOY_3RDPARTY_WHEEL_DIR}/${PANTS_UNSTABLE_VERSION}" \
-    "${requirements[@]}"
+  (
+    PEX_VERSION=$(grep "pex==" "${ROOT}/3rdparty/python/requirements.txt" | sed -e "s|pex==||")
```

Comment:
one potential alternative to this would be:

```
pex --constraints=3rdparty/python/requirements.txt pex
```

which should net the same effect w/o the manual parsing.

Reply:
There's a bootstrap issue here. This grabs the pants pex dep to form the url to fetch the corresponding released pex pex, so we don't have pex yet.